test/integration/patients/patient/dashboard.js

```diff
@@ -299,7 +299,7 @@ context('patient dashboard page', function() {
       .find('[data-owner-region]')
       .find('button')
       .should('be.disabled')
-      .should('contain', 'Clinician M.');
+      .should('contain', 'Clinician McTester');
```

Comment:
I updated these to be the full name.  I know they're not displayed entirely, but that's what's in there.